query/mutation_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build integration || cloud || upgrade
+//go:build integration || upgrade
```

Comment:
We don't want to run them in cloud?

Reply:
We don't run anything in cloud today. We don't use the cloud tag.